modules/core/src/main/scala/org/tessellation/domain/snapshot/SnapshotReference.scala

```diff
@@ -0,0 +1,15 @@
+package org.tessellation.domain.snapshot
+
+import cats.data.NonEmptySet
+
+import org.tessellation.kernel.StateChannelSnapshot
+import org.tessellation.schema.height.Height
+import org.tessellation.schema.peer.PeerId
+
+case class SnapshotReference(
+  ordinal: SnapshotOrdinal,
+  height: Height,
+  subHeight: Height,
+  snapshots: Set[StateChannelSnapshot],
```

Comment:
I think that this doesn't have to be in the `SnapshotReference`. The `GlobalSnapshot` is going to contain state channel snapshot.